package.json

```diff
@@ -3,6 +3,10 @@
     "version": "0.18.2",
     "description": "ReactJS SDK that provides login functionality with SuperTokens.",
     "main": "./index.js",
+    "engines": {
+        "node": ">=16.0.0",
+        "npm": "^8"
+    },
```

Comment:
I don't think we should enforce this. node 14 is still very common. Therefore, we should also remove the change in .npmrc

Reply:
Removed `.npmrc`